src/main/java/com/thevoxelbox/voxelsniper/brush/GenerateTreeBrush.java

```diff
@@ -36,15 +35,12 @@ public class GenerateTreeBrush extends Brush {
     private int thickness = 1;
     private int slopeChance = 40;
     private final int twistChance = 5; // This is a hidden value not available through Parameters. Otherwise messy.
-    private int heightMininmum = 14;
+    private int heightMinimum = 14;
     private int heightMaximum = 18;
```

Comment:
```suggestion
    private int minimumHeight = 14;
    private int maximumHeight = 18;
```

Wouldn't this make more sense? (obviously also change in all other places)